ParkingZone/Areas/Admin/Views/ParkingZones/Create.cshtml

```diff
@@ -1,12 +1,12 @@
-﻿@model global::ParkingZone.Models.ParkingZone
+﻿@model global::ParkingZone.Domain.Entities.ParkingZone
```

Comment:
What does `global::` mean? Could you, please, explain?

Reply:
It's often added to Auto-generated code to avoid from reference conflict. when 'global::'  is used it finds namespaces and 
models .  for example :  u have collar model and u wand to use collar.Red , in that kind of situation u can use global::